hypothesis-python/tests/django/toystore/test_given_models.py

```diff
@@ -52,20 +58,31 @@
     Store,
 )
 
-add_default_field_mapping(CustomishField, just(u"a"))
+register_field_strategy(CustomishField, just(u"a"))
+
+
+class UnregisteredCustomishField(CustomishField):
```

Comment:
nit: Would it make sense to have this class be method local? Right now this creates the slightly weird property (that is shared by many of our tests but no need to perpetuate it) that the test would go from failing to passing if rerun. It doesn't erally matter though.